tests/e2e/testsLibrary/che-code/ProjectAndFileTestsCode.ts

```diff
@@ -0,0 +1,34 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import 'reflect-metadata';
+import { inject, injectable } from 'inversify';
+import { By, until } from 'selenium-webdriver';
+import { DriverHelper } from '../../utils/DriverHelper';
+import { CLASSES } from '../../inversify.types';
+
+@injectable()
+export class ProjectAndFileTestsCode {
+
+    constructor(
+        @inject(CLASSES.DriverHelper) private readonly driverHelper: DriverHelper) {}
+
+    public async waitWorkspaceReadinessForVSCodeEditor(): Promise<void> {
```

Comment:
So that it's officially Che Code editor, the correct method name is `waitWorkspaceReadinessForCheCodeEditor()`.

Reply:
okay, fixed

Reply:
thanks